froide/comments/models.py

```diff
@@ -1,5 +1,17 @@
+from django.db import models
+
 from django_comments.abstracts import CommentAbstractModel
+from django_comments.signals import comment_will_be_posted
 
 
 class FroideComment(CommentAbstractModel):
     ip_address = None
+    comment_as_normal_user = models.BooleanField(null=True)
```

Comment:
I would really like this turned around: `is_moderation` that can be `True`. This way the special handling is emphasized and not the 'normal' handling. Otherwise the question arises when reading: what does not normal mean?
Also I don't think `null=True` makes sense here, I can't see a difference between `False` and `None`.

Reply:
I made it this way around to avoid extensive migrations, but can change it